src/lib/assetBridger/ethBridger.ts

```diff
@@ -164,6 +178,45 @@ export class EthBridger extends AssetBridger<
     return L1TransactionReceipt.monkeyPatchEthDepositWait(tx)
   }
 
+  /**
+   * Deposit ETH from L1 onto a different L2 address
+   * @param params
+   * @returns
+   */
+  public async depositTo(
```

Comment:
For each of the functions that send a transaction we currently offer a `getXRequest` variant of the function, this allows people to create the tx and send it later. We then call this `getXRequest` function from with the version of the func that actually sends the transaction - see ethBridger.deposit / ethBridger.getDepositRequest. We should add a `getDepositToRequest` func and use it this way - you can internally use getTicketCreationRequest

Reply:
Oh, good idea. I'll try to add a getDepositToRequest.